docs_app/src/app/custom-elements/code/code.component.ts

```diff
@@ -161,6 +182,17 @@ export class CodeComponent implements OnChanges {
     }
   }
 
+  editInStackBlitz() {
+    this.stackblitz.openProject({
+      code: this.codeText,
+      language: this.language,
+      dependencies: {
+        rxjs: '6'
```

Comment:
In the final version, this shouldn't be hard-coded, I assume.

Reply:
I mean, it should be somehow populated (either at build time or at runtime) from the current version.